sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/models/BlobQueryOptions.java

```diff
@@ -4,6 +4,7 @@
 package com.azure.storage.blob.models;
 
 import com.azure.core.annotation.Fluent;
+import com.azure.storage.common.implementation.StorageImplUtils;
```

Comment:
This needs to go in the options package per the new guidelines: https://azure.github.io/azure-sdk/java_design.html#service-method-parameters